### PR TITLE
Loading queries example

### LINKED ISSUE #73: make Query Component work with the data structure

### PR DESCRIPTION
**Pull request by NybyDK**
Closes #73 according to Jesper

### Changes/Additions:
- added testings for queries
- connected frontend query to backend structure stuff

### CODE REVIEW

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

**Comment by NybyDK** [comment]
Jeg vil da forhelvede ikke dismiss, jeg vil bare have main ind for 4. gang 😭

File: src/lib/components/query/Query.svelte [hunk 1]
```diff
@@ -79,6 +90,7 @@
 		align-items: center;
 		padding-right: 1em;
 	}
+
```
Comment (BaBrixx) [inline src/lib/components/query/Query.svelte:93]: I cred🥲

File: src/lib/components/query/QueryDropDownMenu.svelte
Comment (BaBrixx) [inline src/lib/components/query/QueryDropDownMenu.svelte:1]: The context menu should disappear when it is already open and one clicks the button again.

**Review by BaBrixx — CHANGES_REQUESTED** [review]

File: src/lib/components/query/Query.svelte (hunk 1 above)
Comment (Jesper276) [reply src/lib/components/query/Query.svelte:93]: i kill

File: src/lib/components/query/QueryDropDownMenu.svelte
Comment (NybyDK) [reply src/lib/components/query/QueryDropDownMenu.svelte:1]: Will be added to ContextMenu itself, will create an issue

**Review by BaBrixx — APPROVED** [review]
LGTM

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

**Pull request merged**